.env

```diff
@@ -0,0 +1,6 @@
+# Don't commit this file with development config to Git
+# DB_HOST= postgres for docker or localhost for development
+DB_HOST=postgres
+DB_USERNAME=mooadmin
+DB_PASSWORD=moopass
+DB_NAME=moodb
```

Comment:
Add a new line

Reply:
addressed. Added a PORT var. which I forgot to commit